src/main/java/step2/model/LottoNumbers.java

```diff
@@ -0,0 +1,30 @@
+package step2.model;
+
+import java.util.List;
+
+public class LottoNumbers {
+
+	private List<Integer> lottoNumbers;
+
+	private LottoNumbers(List<Integer> lottoNumbers) {
+		this.lottoNumbers = lottoNumbers;
+	}
+
+	public static LottoNumbers randomNumbers() {
+		RandomLottoFactory randomLottoFactory = new RandomLottoFactory();
+		return new LottoNumbers(randomLottoFactory.randomLotto());
```

Comment:
`RandomLottoFactory`를 정적 메서드만을 포함하는 유틸리티 클래스로 만들어서 사용해도 괜찮을 것 같아요.
매번 임의의 6개의 번호 묶음을 반환할 때마다 `RandomLottoFactory` 인스턴스를 생성하지 않아도 되지 않을까요?
```suggestion
		return new LottoNumbers(RandomLottoFactory.randomLotto());
```